### PR TITLE
Add a changelog

### LINKED ISSUE #63: Add a changelog
Add a changelog according to: https://keepachangelog.com/en/1.0.0/
I suggest tracking changes from version 1.0.0 😄

### PR DESCRIPTION
**Pull request by svenvanderburg**
To be updated when we do the full 1.0.0 release.
Fixes #63

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- `nanopub.client` module with the NanopubClient class that implements:
+  * Searching (being a client with a direct (but incomplete) mapping to the nanopub server grlc endpoint):
+    * `find_nanopubs_with_text` method
+    * `find_nanopubs_with_pattern` method
+    * `find_things` method
+  * Fetching:
+    * `fetch` method to fetch a nanopublication
+  * Publishing:
+    * Publish a statement using `claim` method
+    * Publish a `Publication` object with `publish` method
```
Comment (raar1) [inline CHANGELOG.md:21]:
```suggestion
    * Publish a `Publication` object with `publish` method
  * Retracting:
    * Publish a retraction of an existing nanopublication created by this user (i.e. signed with same RSA key)
  
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- `nanopub.publication` module
```
Comment (raar1) [inline CHANGELOG.md:22]:
```suggestion
  * Test server functionality
    * Client can optionally be set to publish to (and fetch from) the nanopub test servers only.

- `nanopub.publication` module
```

**Review by raar1 — APPROVED** [review]
Looks good. I made some suggested commits mentioning the retraction and test server functionality.

**Comment by svenvanderburg** [comment]
Nice additions! I totally missed those...

**Pull request merged**